docs/27-Funcs_Analysis.md

```diff
@@ -1421,11 +1421,12 @@ to Calc's solvers.
 
 There are three constraint inequalities:
 
-```java
-x <= 6
-y <= 8
-z >= 4
-```
+=== "java"
+    ```java
+    x <= 6
+    y <= 8
+    z <= 4
```

Comment:
Is this change (z >=4 -> z <=4 ) intentional?

Reply:
No, manual conflict resolution instead of scripted change.